DataFormats/HcalCalibObjects/src/classes_def.xml

```diff
@@ -3,6 +3,9 @@
    <version ClassVersion="16" checksum="2161587371"/>
    <version ClassVersion="17" checksum="1351689277"/>
   </class>
+   <ioread sourceClass="HOCalibVariables" targetClass="HOCalibVariables" version="[-16]" target="pileup" source="double insLumi" embed="false">
+    <![CDATA[ pileup = onfile.insLumi; ]]>
+  </ioread>
```

Comment:
```suggestion
   <ioread sourceClass="HOCalibVariables" targetClass="HOCalibVariables" version="[-16]" target="pileup" source="float inslumi" embed="false">
    <![CDATA[ pileup = onfile.inslumi; ]]>
  </ioread>
```
I'd assume ioread rules are case-sensitive (also, the type was `float`).
https://github.com/bsunanda/cmssw/commit/ef5ad54a923bb574888b3373357cd931ab441fd6#diff-05559b5e9d212b1045ba8feaed01c0f7b24c190f79b82807e02cadb0e052c2edL10